orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltOrchestrator.java

```diff
@@ -141,6 +141,9 @@
     @Value("${cb.max.salt.database.dr.retry.onerror:5}")
     private int maxDatabaseDrRetryOnError;
 
+    @Value("${cb.max.salt.validation.retry:3}")
+    private int maxValidationRetry;
```

Comment:
i think this should be a bit more specific as that is only used for storage validation